test/integration/resultsPage.js

```diff
@@ -18,6 +18,51 @@ function assertSearchResponse(search, assertions) {
     .end(assertions);
 }
 
+function searchNameRanking(res, expected) {
+  const $ = cheerio.load(res.text);
+  const searchResultArr = [];
```

Comment:
It is best to try not to name variables after the type to prevent them having to be renamed if the type changes. `searchResults` would be ok here (and in the below code)